crates/node/core/src/node_config.rs

```diff
@@ -395,8 +400,9 @@ impl<ChainSpec> NodeConfig<ChainSpec> {
     /// Change rpc port numbers based on the instance number, using the inner
     /// [`RpcServerArgs::adjust_instance_ports`] method.
     pub fn adjust_instance_ports(&mut self) {
-        self.rpc.adjust_instance_ports(self.instance);
-        self.network.adjust_instance_ports(self.instance);
+        let instance = self.get_instance();
+        self.rpc.adjust_instance_ports(instance);
```

Comment:
what we should do here now is extract this to a standalone fn

https://github.com/paradigmxyz/reth/blob/85f66ff7ac19732af623234a0c945b002919a585/crates/node/core/src/args/rpc_server.rs#L250-L253

so that the can handle the Non vs Some(1) case, check the referenced issue